hudi-client/hudi-spark-client/src/main/scala/org/apache/spark/HoodieSparkKryoRegistrar.scala

```diff
@@ -61,6 +62,7 @@ class HoodieSparkKryoRegistrar extends HoodieCommonKryoRegistrar with KryoRegist
     kryo.register(classOf[HoodieInternalRow])
     kryo.register(classOf[HoodieSparkCopyOnWriteTable[_]])
     kryo.register(classOf[HoodieSparkMergeOnReadTable[_]])
+    kryo.register(classOf[HoodieKeyLookupResult])
```

Comment:
Previously the new classes are added at the end but I think this is OK since the Kryo serde is only used in memory, not for any storage bytes any more.